un-climate-change.csl

```diff
@@ -0,0 +1,238 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" default-locale="en-GB" page-range-format="expanded">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>UN Climate Change</title>
+    <title-short>UNFCCC</title-short>
+    <id>http://www.zotero.org/styles/un-climate-change</id>
+    <link href="http://www.zotero.org/styles/un-climate-change" rel="self"/>
+    <link href="https://www.zotero.org/styles/harvard-kings-college-london" rel="template"/>
```

Comment:
@mserifovic, "https://www.zotero.org/styles/harvard-kings-college-london" should be "http://www.zotero.org/styles/harvard-kings-college-london" (with "http" instead of "https"). That's the only issue right now in the automatic testing.